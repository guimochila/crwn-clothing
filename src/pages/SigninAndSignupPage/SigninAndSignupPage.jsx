import React from 'react';
import './SigninAndSignupPage.scss';
import Signin from '../../components/Signin';
import Signup from '../../components/Signup';

function SigninAndSignupPage() {
  return (
    <div className="sign-in-and-sign-up">
      <Signin />
      <Signup />
    </div>
  );
}

export default SigninAndSignupPage;
