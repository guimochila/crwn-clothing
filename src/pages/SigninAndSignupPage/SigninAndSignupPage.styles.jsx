import styled from 'styled-components';

export const SigninAndSignupContainer = styled.div`
  width: 850px;
  display: flex;
  justify-content: space-between;
  margin: 30px auto;

  @media screen and (max-width: 800px) {
    width: 100%;
    display: grid;
    grid-template-columns: 1fr;
    grid-gap: 60px;
  }
`;
