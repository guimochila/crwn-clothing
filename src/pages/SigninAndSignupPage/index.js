import SigninAndSignupPage from './SigninAndSignupPage';

export default SigninAndSignupPage;
