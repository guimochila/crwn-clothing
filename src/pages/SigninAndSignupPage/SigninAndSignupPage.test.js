import React from 'react';
import { shallow } from 'enzyme';

import SigninAndSignupPage from './SigninAndSignupPage';

describe('Signup and SignIn page', () => {
  test('should render SigninAndSignup page', () => {
    expect(shallow(<SigninAndSignupPage />)).toMatchSnapshot();
  });
});
