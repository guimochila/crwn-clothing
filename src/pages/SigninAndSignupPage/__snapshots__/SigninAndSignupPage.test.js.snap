// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signup and SignIn page should render SigninAndSignup page 1`] = `
<styled.div>
  <Connect(Signin) />
  <Connect(Signup) />
</styled.div>
`;
